repo.json

```diff
@@ -6,6 +6,6 @@
     "additional_info": "See the README.md at the root of this repository for more information",
     "last_updated": "2022-09-27",
     "templates": [
-        "https://raw.githubusercontent.com/o3de/o3de-extras/development/Templates/Multiplayer"
+        "https://github.com/o3de/o3de-extras/tree/development/Templates/Multiplayer"
```

Comment:
not sure this is correct either, but for now ok.

Reply:
Fixed! @AMZN-Phil caught this as well. 
Updated the template link to use .git; the original returns the page you'd see in a web-browser, whereas .git causes the script to use the git API to get the raw file